### ISSUE TITLE
Confusing message: Can I remove /etc/fstab?

### ISSUE THREAD

**Issue by geonnave** [NixOS/nix#9310]
## Platform

- [ ] Linux: 
- [x] macOS
- [ ] WSL

## Additional information

I am trying to install `nix` by following the [website instructions for MacOS](https://nixos.org/download#nix-install-macos).

During installation, it asks me: `Can I remove /etc/fstab?`

The problem is, I don't know what /etc/fstab is... so I googled a bit and it seems it might be dangerous to do so? Or maybe it is not? Why nix needs to remove it?

So my issue is just this, that the question is confusing and I don't know if I should allow that.

## Output

<details><summary>Output</summary>

```log
$ sh <(curl -L https://nixos.org/nix/install)

(...)

During install, I add '/nix' to /etc/fstab so that macOS knows what
mount options to use for the Nix volume.
Can I remove /etc/fstab?
[y/n]
```

</details>

## Priorities

Add :+1: to [issues you find important](https://github.com/NixOS/nix/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).

**Comment by abathur** [comment]
What's in the file?

**Comment by geonnave** [comment]
```
$ cat /etc/fstab
#
# Warning - this file should only be modified with vifs(8)
#
# Failure to do so is unsupported and may be destructive.
#
UUID=1AF1DF87-A629-471E-B0B7-FB90FD4EA2B0 /nix apfs rw,noauto,nobrowse,suid,owners
```

Looks like it is some nix related config, although fstab itself seems to be more generic and _could_ contain other items.

**Comment by abathur** [comment]
Correct. So, you've already got (or previously had) Nix installed.

It would be okay to give consent to the prompt, but if you already had Nix installed, there are probably more important questions here.

If you're trying to reinstall, why? Is this a shared system?

**Comment by geonnave** [comment]
I tried to reinstall because:
- the package I was trying to install (`nix profile install github:hacspec/hax`) was failing, and after many retries, I decided to maybe re-install nix
- it is not a shared system, it is my personal mac computer

In the end, I went ahead and deleted the `/etc/fstab` file. From my point of view, the issue can be closed. As a suggestion, the installer might link to a resource where the meaning of this file is explained.
